openpype/hosts/maya/api/plugin.py

```diff
@@ -523,6 +523,49 @@ def remove_instances(self, instances):
 class Loader(LoaderPlugin):
     hosts = ["maya"]
 
+    def get_custom_namespace_and_group(self, context, options, loader_key):
+        """Queries Settings to get custom template for namespace and group.
+
+        Group template might be empty >> this forces to not wrap imported items
+        into separate group.
+
+        Args:
+            context (dict)
+            options (dict): artist modifiable options from dialog
+            loader_key (str): key to get separate configuration from Settings
+                ('reference_loader'|'import_loader')
+        """
+        asset = context['asset']
+        subset = context['subset']
+        settings = get_project_settings(context['project']['name'])
+        custom_naming = settings['maya']['load'][loader_key]
+        options["attach_to_root"] = True
+        if not custom_naming['namespace']:
+            raise LoadError("No namespace specified in "
+                            "Maya ReferenceLoader settings")
+        elif not custom_naming['group_name']:
+            self.log.debug("No custom group_name, no group will be created.")
+            options["attach_to_root"] = False
+        formatting_data = {
+            "asset_name": asset['name'],
+            "asset_type": asset['type'],
+            "folder": {
+                "name": asset["name"],
+            },
+            "subset": subset['name'],
+            "family": (
+                    subset['data'].get('family') or
+                    subset['data']['families'][0]
+            )
+        }
+        custom_namespace = custom_naming['namespace'].format(
+            **formatting_data
+        )
+        custom_group_name = custom_naming['group_name'].format(
+            **formatting_data
+        )
+        return custom_group_name, custom_namespace, options
```

Comment:
To me personally the removed empty lines here in the code actually make this function much harder to read than the original. It's cosmetics and it is an opinion. Just wanted to remark that reading this version feels much more cluttered to me than the original. So if you have no preference, let's keep the original. If you prefer this, you're a maniac - haha, no, do whatever you want. 😈

Reply:
Hmm, this is totally unwanted, probably copy&paste issue. Will fix that.

Reply:
Btw this is client requested, see `sponsored` tag on this PR which denotes that some customer requested that. Who am I to argue :D.